index.src.html

```diff
@@ -459,6 +479,28 @@
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  ## The `CredentialCreationData` Dictionary ## {#credentialcreationdata-dictionary}
+
+  In order to create a {{Credential}} via {{CredentialsContainer/create()}}, the caller specifies a
+  few parameters in a {{CredentialCreationData}} object.
+
+  Note: The {{CredentialCreationData}} dictionary is an extension point. If and when new types of
+  credentials are introduced, they will add to the dictionary so they can be passed into the
+  creation method. See [[#implementation-extension]].
+
+  <pre class="idl">
+    [Exposed=Window, SecureContext]
+    dictionary CredentialCreationData {
+      boolean unmediated = false;
+    };
+  </pre>
+  <dl dfn-for="CredentialCreationData">
+    :   <dfn dict-member>unmediated</dfn>
+    ::  If `true`, the user agent will only attempt to create a {{Credential}} if it can be done
```

Comment:
Well, the notion here is that unmediated wouldn't store the credential, but just give you an object. Similar to `new PasswordCredential`. Of course, we don't define `mediated` yet, so it's basically a no-op until such time as the browser learns to do new ui.